### ISSUE TITLE
add-path command is disabled

### ISSUE THREAD

**Issue by Hyzual** [e1himself/goss-installation-action#4]
Hello,
First of all, thanks for making this github action, it makes it really easy to install dgoss and run tests on docker containers :).

It looks like the add-path command has been removed by Github due to security concerns. Running this action in a pipeline gives the following warnings :

```
Unable to process command '::add-path::/opt/hostedtoolcache/goss/0.3.13/amd64' successfully.
The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

I'll try to fork the repo and see if I can come up with a fix

**Comment by Hyzual** [comment]
It looks like the fix is easy: 

(from https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/#patching-your-actions-and-workflows)
> Action authors who are using the toolkit should update the @actions/core package to v1.2.6 or greater to get the updated addPath and exportVariable functions.

The fix is merging #3

**Comment by e1himself** [comment]
Thanks for your help here @Hyzual :bow: 

I've just merged and released #3.

Please confirm the issue is fixed.

Cheers! :wink:

**Comment by Hyzual** [comment]
@e1himself It works fine, my workflow ran without problem. Thanks a lot for the quick response and the release :smile: 
I confirm the issue is fixed.
Cheers !
